exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/OtelToZipkinSpanTransformerTest.java

```diff
@@ -138,7 +133,7 @@ void generateSpan_ProducerKind() {
   @Test
   void generateSpan_ResourceServiceNameMapping() {
     Resource resource =
-        Resource.create(Attributes.of(ResourceAttributes.SERVICE_NAME, "super-zipkin-service"));
+        Resource.create(Attributes.of(stringKey("service.name"), "super-zipkin-service"));
```

Comment:
I really fail to see how this can be an improvement.

Reply:
There is an artifact with semantic conventions: `io.opentelemetry.semconv:opentelemetry-semconv:1.21.0-alpha`

There are a few semantic conventions which are stable, but the tooling for `semantic-conventions-java` (previously in this repo but deprecated as of this PR) isn't currently able to differentiate between stable / unstable. As a consequence, the whole artifact is marked as alpha, and has the ability to make breaking changes to the code generation logic. 

If the code in this repository adds a dependency on `io.opentelemetyr.semconv:opentelemetry-semconv` and imports a constant, then later an (allowed) breaking change occurs which moves the location of that constant, users will have to deal with a version conflict:
- app depends on `opentelemetry-exporter-zipkin:1.30.0` which depends on `opentelemetry-semconv:1.21.0-alpha`
- app depends on `opentelemetry-semconv:1.25.0`, which made a breaking change to the code generation logic
- user must choose to upgrade to a later version of `opentelemetry-exporter-zipkin` which shares the `opentelemetry-semconv:1.25.0` version, or downgrade `opentelemetry-semconv` to `1.21.0-alpha`. 

This wouldn't be a problem is `io.opentelemetry.semconv:opentelemetry-semconv` is stable, but there are open problems to work through before thats possible (see [#22](https://github.com/open-telemetry/semantic-conventions-java/issues/22), [#6](https://github.com/open-telemetry/semantic-conventions-java/issues/6), [#5](https://github.com/open-telemetry/semantic-conventions-java/issues/5)).

When there is a stable artifact, we can consume that in this repository without worrying that we'll create potential dependency conflicts in the future. But for now, its perfectly safe for these repos to inline the attributes they need. They're just strings after all.

Reply:
It's just that I had recently replaced all the string in Quarkus by these conventions and now they will vanish... 
I guess I need to get the conventions anyway from somewhere.
Using strings for this can lead to insidious bugs.